util/og_api_postman_collection.json

```diff
@@ -990,6 +990,48 @@
 					},
 					"response": []
 				},
+				{
+					"name": "Get Crosshair Inensity",
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{og-access-token}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{og-api}}/images/20230605080000/CM-202-CVC-CAM-1/crosshair?functions={\"name\": \"a\", \"expression\": \"CM-202-CVC-CAM-1 %2B 1280\"}",
+							"host": [
+								"{{og-api}}"
+							],
+							"path": [
+								"images",
+								"20230605080000",
+								"CM-202-CVC-CAM-1",
+								"crosshair"
+							],
+							"query": [
+								{
+									"key": "position",
+									"value": "[600, 600]",
```

Comment:
It would be better if the x and y position values were different so that it is easier to correlate them with the values in the returned json.